src/component/management/profile/ProfileStyles.less

```diff
@@ -80,36 +84,37 @@ body {
 }
 
 .profile-label{
-    font-size: 20px; 
-    display: inline; 
+    font-size: 20px;
+    display: inline;
+    padding-top: 10;
 }
 
 #password-change{
     margin-top: 10px;
 }
 
 .profile-row{
-    border-bottom: 1px solid #17161d; 
+    border-bottom: 1px solid #17161d;
     padding: 20px 0px 0px 5px;
     &:hover{
-        background: #4d4966; 
-        cursor: pointer; 
+        background: #4d4966;
+        cursor: pointer;
     }
 }
 
 .exitOverlay{
-  background: rgba(0, 0, 0, 0.95); 
+  background: rgba(0, 0, 0, 0.95);
```

Comment:
Keep four spaces for tab space here.
Correct for below lines as well.

Reply:
What's the differences here? Should I add a space after each line?